litmus-portal/frontend/src/views/Settings/UserManagementTab/EditUser/DelUser/index.tsx

```diff
@@ -32,9 +32,9 @@ const DelUser: React.FC<DelUserProps> = ({
 
   return (
     <div>
-      {/* {tableDelete ? (
+      {tableDelete ? (
         <>
-          <MenuItem
+          {/*  <MenuItem
```

Comment:
comments can be removed

Reply:
The backend is not there that's why we had to comment it fo now